Fix a small typo in manpage

An option is spelled incorrectly.
--- a/docs/eyeD3.1
+++ b/docs/eyeD3.1
@@ -249,7 +249,7 @@
 \fB--remove-all-images
 Remove all images from the tag
 .TP
-\fB--write--images=\fRDIR
+\fB--write-images=\fRDIR
 Causes all attached images (APIC frames) to be written to the specified directory.
 .TP
 \fB--add-object=\fROBJ_PATH:MIME-TYPE[:DESCRIPTION[:FILENAME]]
